docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+
+
+Teleport Access Monitoring allows to query and analyze the audit events generated by Teleport.
+It provides a SQL interface to query the audit logs and generate reports.
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
```

Comment:
The first 3 sentences of this paragraph all say basically the same thing: access monitoring allows you to query and analyze audit logs.

Can we remove some of the duplication here and simplify the language?

Reply:
squeezed to: 

> Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.